packages/InteractiveProfilingTool-Tests.package/DummyClass.class/instance/dummyMethodToProfile.st

```diff
@@ -0,0 +1,3 @@
+as yet unclassified
+dummyMethodToProfile
+   	^ 1
```

Comment:
Please consider adding a newline after the function name to be consistent with other methods.